tkweb/apps/drinks/templates/drinks/home.html

```diff
@@ -0,0 +1,12 @@
+{% extends "drinks/base.html" %}
+{% block fulltitle %}Drinks{% endblock %}
+{% block content %}
+<h1>Barkort generator:</h1>
+<p> Vælg det barkort, som du ønsker at generere </p>
+<form action="generate/" method="post">
```

Comment:
Her bør du bruge `{% url "generate" %}` til at lave linket i stedet for at skrive `generate/` eksplicit. Så slipper vi for at skrive den samme URL i flere filer, og så er det nemt at ændre URL'er hvis vi vil gøre det.